packages/wallet/backend/src/user/service.ts

```diff
@@ -110,6 +117,66 @@ export class UserService implements IUserService {
     return !!user
   }
 
+  public async createDefaultAccount(email: string) {
+    const existingUser = await this.getByEmail(email)
+
+    if (existingUser) return
+
+    let defaultUser = this.env.DEFAULT_WALLET_ACCOUNT
```

Comment:
`const defaultUser = email === this.env.DEFAULT_BOUTIQUE_ACCOUNT.email ? this.env.DEFAULT_BOUTIQUE_ACCOUNT : this.env.DEFAULT_WALLET_ACCOUNT`